tests/test_metsdoc.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import StringIO
+
+from pymets import metsdoc
+
+
+class METSDocTests(unittest.TestCase):
+
+    def test_invalid_mets_element(self):
+
+        mets_string = """<?xml version="1.0" encoding="UTF-8"?>
+        <mets><metsHDR/></mets>"""
```

Comment:
Since the point of the triple quotes in this case is to maintain the string's structure (linebreak), maybe it would be better to display how the string looks more accurately? I'm thinking something like this: